File: benchmarks/live_suite/fib.py
def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

if __name__ == "__main__":
    res = fib(35) # Note: fib(40) in pure python takes ~15s, so we test fib(35) and scale, or test fib(35)
    print(f"Result: {res}")

File: benchmarks/live_suite/fib.rs
fn fib(n: u64) -> u64 {
    if n <= 1 {
        n
    } else {
        fib(n - 1) + fib(n - 2)
    }
}

fn main() {
    let res = fib(40);
    println!("Result: {}", res);
}
